beacon-chain/db/kv/operations.go

```diff
@@ -13,32 +13,23 @@ import (
 func (kv *Store) VoluntaryExit(ctx context.Context, exitRoot [32]byte) (*ethpb.VoluntaryExit, error) {
 	ctx, span := trace.StartSpan(ctx, "BeaconDB.VoluntaryExit")
 	defer span.End()
-	var exit *ethpb.VoluntaryExit
-	err := kv.db.View(func(tx *bolt.Tx) error {
-		bkt := tx.Bucket(voluntaryExitsBucket)
-		enc := bkt.Get(exitRoot[:])
-		if enc == nil {
-			return nil
-		}
-		exit = &ethpb.VoluntaryExit{}
-		return decode(ctx, enc, exit)
-	})
-	return exit, err
+	enc, err := kv.voluntaryExitBytes(ctx, exitRoot)
+	if err != nil {
+		return nil, err
+	}
+	exit := &ethpb.VoluntaryExit{}
+	return exit, decode(ctx, enc, exit)
```

Comment:
it looks off decoding it like this, can you decode it above and then only return the exit.